packages/core/src/styles/layout/docs/layout.stories.mdx

```diff
@@ -86,20 +86,25 @@ manage layouts on many different screen sizes. All responsive utilities follow t
 
 <Story id="foundation-design-tokens-stories--layout" />
 
-```css
+```scss
 @media (min-width: 576px) {
```

Comment:
We may want to consider setting these sizes in SASS variables or a SASS map like we used to so people can get to them more easily. Preferably they could be SASS variables generated by tokens.yml.

Out of scope with what you're doing here. But plz open a :octocat: issue if you agree.

Reply:
We do have them in the tokens file and a sass map here
https://github.com/vmware/clarity/blob/master/packages/core/tokens.yml#L111 

The map is using the generated vars from the tokens file
https://github.com/vmware/clarity/blob/master/packages/core/src/styles/layout/mixins/_mixins.scss#L16 

I think this was here for documentation on what the breakpoint values were since we cant use custom props in the media queries.